# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/e17/examine/gcc34.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./src/examine_client.c.vanilla	2004-06-21 22:08:42.991214496 +0200
+++ ./src/examine_client.c	2004-06-21 22:09:21.631340304 +0200
@@ -25,6 +25,9 @@
 Ecore_Config_Ipc_Server_List *examine_client_server;
 examine_prop   *prop_list;
 
+void examine_client_list_props_cb(void);
+void examine_client_get_val_cb(void);
+
 /*****************************************************************************/
 
 
